solutions/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/org-policies/exceptions/gcp-resource-locations-except-host-project.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+#
+# GCP Organization Policies
+# Org policies that correspond with a Guardrail will contain a label indicating what Guardrails it helps in enforcing
+# https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints
+#
+# Constraint: constraints/gcp.resourceLocations
+#
+# This list constraint defines the set of locations where location-based GCP resources can be created.
+#
+# This exception is to allow the us-central1 location until the issue below is resolved with the config connector forwardingRule resource.
+# This resource is used by the PSC for googleapis.
+# https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/754
+#
+#########
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: ResourceManagerPolicy
+metadata:
+  name: gcp-restrict-resource-locations-except-host-project-id # kpt-set: gcp-restrict-resource-locations-except-${host-project-id}
+  namespace: policies
+  labels:
+    guardrail: "true"
+    guardrails-enforced: guardrail-05
+  annotations:
+    config.kubernetes.io/depends-on: resourcemanager.cnrm.cloud.google.com/namespaces/client-name-projects/Project/host-project-id # kpt-set: resourcemanager.cnrm.cloud.google.com/namespaces/${client-name}-projects/Project/${host-project-id}
+spec:
+  constraint: "constraints/gcp.resourceLocations"
+  listPolicy:
+    allow:
+      values:
+        - "in:northamerica-northeast1-locations"
+        - "in:northamerica-northeast2-locations"
+        - "in:us-central1-locations"
```

Comment:
Wanted to check that us-central1 is there on purpose?

Reply:
Makes sense! /approve

Reply:
from my understanding, it's supposed to be temporary until the bug describe on line 24 is resolved...

@davelanglois-ssc can confirm?